docs/sftt/sftt-setup-backend.md

```diff
@@ -0,0 +1,32 @@
+# Setting up the Backend
+This is a comprehensive overview of the steps required and everything needed to run the database locally. 
+
+### Downloads
+1. Clone the [Github repo](https://github.com/Code-4-Community/speak-for-the-trees-backend)
+2. Download [PostgreSQL](https://www.postgresql.org/download/) (Make sure it comes with pgAdmin4)
+3. Download [Java JDK 8](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
+4. Download [Java JRE 8](https://www.oracle.com/java/technologies/javase-jre8-downloads.html)
+5. Download [IntelliJ](https://www.jetbrains.com/idea/download/) (Community is fine, Ultimate available through student account)
+
+### Installing Maven
+1. Download [Maven](https://maven.apache.org/download.cgi)
+2. Follow the directions on the [install page](https://maven.apache.org/install.html)
+3. Helpful guide on setting environment variables for Windows: [link](https://mkyong.com/java/how-to-set-java_home-on-windows-10/)
+4. Make sure the JAVA_HOME enviroment variable is set to the recently downloaded Java JDK 8
+
+### Start a Local Postgres Database
+1. Start pgAdmin4
+2. A local database should start automatically, by default on port 5432
+
```

Comment:
Replace step 2 with that they need to create a database named exactly 'speak-for-the-trees' or 'lucys-love-bus'. If they don't, the scripts will look for a database of one of those names during 'mvn clean install' and throw an error. They don't need to create a user for those databases though. Just using the postgres super user login in the db.properties file should work.